### ISSUE TITLE
13.5.1 (13.3/2) Missing comma

### ISSUE THREAD

**Issue by beemirang** [pat-rogers/Ada-202x-WG9-Informal-Review#110]
"the component_clauses for which the value of last_bit is greater than or equal to the size of the largest machine scalar{, } directly specify the position and size of the corresponding component;"

**Comment by JohnGPBarnes** [comment]
It is a restrictive relative clause which according to Fowler does not require a comma.

**Comment by ARG-Editor** [comment]
I agree with John (and trust his judgement on these sorts of issues), so no action will be taken on this issue.
